ocaml/src/osd_bench.ml

```diff
@@ -54,6 +54,24 @@ let gets (client: Osd.osd) progress n value_size period prefix =
   measured_loop progress do_one n >>= fun r ->
   report "gets" r
 
+
+let partial_reads (client : Osd.osd) progress n _value_size partial_fetch_size period prefix =
+  let gen = make_key period prefix in
+  let target = Lwt_bytes.create partial_fetch_size in
+  let do_one i =
+    let key = gen () in
+    client # partial_get
+           Osd.High
+           (Slice.wrap_string key)
+           [ 0, partial_fetch_size, target, 0 ] >>= function
```

Comment:
I don't think the offset really matters for the performance, so that's why I left it out